srcpkgs/m4ri/template

```diff
@@ -0,0 +1,32 @@
+# Template file for 'm4ri'
+pkgname=m4ri
+version=20200125
+revision=1
+build_style=gnu-configure
+configure_args="--enable-openmp"
+hostmakedepends=pkg-config
+makedepends=libpng-devel
+short_desc="Library for fast arithmetic with dense matrices over GF(2)"
+maintainer="Eloi Torrents <eloitorrentsj@gmail.com>"
+license="GPL-2.0-or-later"
+homepage="https://bitbucket.org/malb/m4ri"
+distfiles="https://bitbucket.org/malb/m4ri/downloads/${pkgname}-${version}.tar.gz"
+checksum=0dfb34aed351882a0f2281535ea6f81c690a5efeb14edab131d9ba0dffe44863
+
+build_options="native_build"
+
+if [ -z "$build_option_native_build" ]; then
+	# this disables sse3
```

Comment:
is this a typo, or redundant?

Reply:
IIRC, there is no flag to disable sse3; `-msse3` is enabled based on cpu testing. However, if you disable sse2 then this disables sse3 as well. Moreover, this will not disable sse2 on x86_64 since it's baseline for that arch.

The comment might be mine, I'm usually too terse...

See https://github.com/void-linux/void-packages/pull/32601#discussion_r701444202

Reply:
You don't need to disable SSE2 on i686, we assume it's there.

According to a quick review of the code only SSE2 is used?

Reply:
Aha, the fear is getting `-msse3`... perhaps use this patch instead: https://salsa.debian.org/science-team/libm4ri/-/blob/master/debian/patches/disable_sse3.patch